.env.example

```diff
@@ -16,6 +16,12 @@ PG_USER=<db username>
 PG_HOST=<db host>
 PG_PASS=<db password>
 
+
+#DATABASE CONFIG using string
+# If you intent you use the string please use the below format
```

Comment:
* if the connection does not support ssl

Reply:
Have made the corrections.